docs/core/tools/troubleshoot-dotnet-tool.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Troubleshoot .NET Core tool usage issues
+description: Discover the common issues when running .NET Core tools and possible solutions.
+author: kdollard
+ms.date: 09/23/2019
+---
+
+# Troubleshoot .NET Core tool usage issues
+
+You might come across issues when trying to install or run a .NET Core tool, which can be a global tool or a local tool. This article describes the common root causes and some possible solutions.
+
+## Installed .NET Core tool fails to run
+
+When a .NET Core tool fails to run, most likely you ran into one of the following issues:
+
+* The executable for the tool wasn't found.
+* The correct version of the .NET Core runtime wasn't found. 
+
+### Command or file wasn't found
+
+If the executable file isn't found, you'll see a message similar to:
+
+```
+Could not execute because the specified command or file was not found.
+Possible reasons for this include:
+  * You misspelled a built-in dotnet command.
+  * You intended to execute a .NET Core program, but dotnet-xyz does not exist.
+  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
+```
+
+The executable for the tool may be named in the form `dotnet-<toolName>.exe`, in which case it can be called using `dotnet <toolName>`. It also may be in the form `<toolName>.exe`, in which case it's called using `<toolName>`. The executable may have been created by the tool author (for signed packaged shims) or when the tool is installed (not-signed).
+
+If you're trying to run a .NET Core tool that was installed as a global tool, check that the `PATH` environment variable on your machine contains the value `%USERPROFILE%\.dotnet\tools` on Windows or `$HOME/.dotnet/tools` on macOS and Linux, and that the executable is in that path.
+
+If you're trying to run a .NET Core tool that was installed as a local tool, verify that there's a manifest file called *dotnet-tools.json* or *dotnet-tools.json* in the current or parent directories under a folder named *.config*. If there is such a file, open it and check for the tool you're trying to run. If the file doesn't contain an entry for `"isRoot": true`, then also check further up the file hierarchy for additional tool manifest files.
```

Comment:
> *dotnet-tools.json* or *dotnet-tools.json*

This is the same file name.

Reply:
Good catch! @wli3 did you mean to say that the dotnet-tools.json file can live in the current directory, a parent directory or a child folder named .config?

Reply:
The file need to be called `dotnet-tools.json`. It could live in the current directory, or any parent directories. If it is inside a directory called `.config` we will treat it as if it was not in the `.config` file.

basically the user need to search the following place
```
dotnet-tools.json
.config\dotnet-tools.json
..\dotnet-tools.json
..\.config\dotnet-tools.json
..\..\dotnet-tools.json
..\..\.config\dotnet-tools.json
and on...
```

Reply:
Maybe create a session to explain that?